### PR TITLE
feat(themes): 💬 Support `salla-reviews` `salla-breadcrumb` components

### PR DESCRIPTION
**Pull request by Nabute**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* Code refactor

What is the current behaviour? (You can also link to an open issue here)

* Twig with html implementation

What is the new behaviour? (You can also link to the ticket here)

* Twilight component

Does this PR introduce a breaking change?

* No

Screenshots (If appropriate)

*

### CODE REVIEW

File: src/views/components/home/testimonials.twig
Comment (ashrafreda) [inline src/views/components/home/testimonials.twig:1]: You should cleanup the related css from theme files 
image

**Comment by SallaDev** [comment]
> The branch `feature/TR-123-salla-review-component` of https://github.com/SallaApp/theme-raed/pull/343 has been synced from master..✅

**Review by jalmatari — APPROVED** [review]

**Comment by SallaDev** [comment]
https://github.com/SallaApp/theme-raed/pull/343 is been merged ✅

-------------------

Merge by: @jalmatari

**Pull request merged**